modules/calibration/data/dev_kit_advanced_3lidar/camera_params/front_6mm_extrinsics.yaml

```diff
@@ -0,0 +1,17 @@
+header:
+  seq: 0
+  frame_id: velodyne128
```

Comment:
Just to make sure: so D-kit now has velodyne128 installed as well?

Reply:
It's not a velodyne128 lidar. It is a name to finding correct transform coordinates.

Reply:
Then this might needs to be changed, because the 128 topic might not be available.

Reply:
This one is aligned to dev_kit configuration and has been tested.

Reply:
> This one is aligned to dev_kit configuration and has been tested.

Then the correct code practice is to set the right conf so it is aligned with the installed with the new vehicle and sensor setup, instead of naming the frame_id with sensors that do not installed. This would cause confusions and affect code maintains.

Reply:
cause some reasons of perception model, for now only set framd_id  velodyne128 lidar perception could work  for 
 d-kit ,  this is tricky, I think we can solve this problem later